CONTRIBUTOR_LADDER.md

```diff
@@ -19,6 +19,13 @@ Each of the contributor roles below is organized into lists of three types of th
 
 Open Cluster Management (OCM) is a project made up of several Subprojects, such as Foundation and Policy.  This contributor ladder applies to each Subproject.  As such, a contributor advances in each Subproject separately, and may be a Member in one Subproject while being a Maintainer in another.  For the Open Cluster Management project overall, the contributor is considered to be the highest level they have reached in any Subproject.  Thus, someone who is a Project Member in one Subproject is a Project Member of OCM overall.
 
+| Role | Responsibilities | Requirements | Privileges |
+| ---- | ---------------- | ------------ | ---------- |
+| [Contributor](#contributor) | Casual contributor to the project | n/a | Invitations to contributor events, Eligible to become an Organization Member |
+| [Organization Member](#organization-member) | Regular active contributor in the community | Has pushed at least one PR to an Istio repository | Can recommend a contributor to become a member |
+| [Reviewer](#reviewer) | Hold expertise of a repo or a specific area | Need to review PRs | Can recommend and a member to become a reviewer |
+| [Maintainer](#maintainer) | Reviewing at least 20 PRs per year, especially PRs that involve multiple parts of the project, Participating in, and leading, community meetings | Experience as a Reviewer for at least 3 months, Can commit to spending at least 15 hours per month working on the project | Added to root OWNERs file in one or more Subprojects, Have a vote in Maintainer decision-making meetings and recommand a reviewer to become a maintainer |
```

Comment:
we do not have this Maintainer decision-making meetings :)

Reply:
root OWNERs is not correct, we have repos that owners are in the sub directory which means they are the maintainer of a certain subsystem.

Reply:
Updated.